js-apps/meep-frontend/src/js/containers/meep-container.js

```diff
@@ -27,7 +27,7 @@ import * as meepSandboxCtrlRestApiClient from '../../../../../js-packages/meep-s
 import * as meepMonEngineRestApiClient from '../../../../../js-packages/meep-mon-engine-client/src/index.js';
 import * as meepGisEngineRestApiClient from '../../../../../js-packages/meep-gis-engine-client/src/index.js';
 
-import MeepDrawer from './meep-drawer';
+// import MeepDrawer from './meep-drawer';
```

Comment:
General comment: no need to comment out old code unless it is required for informational purposes to clarify a change; otherwise just remove the old code.

Reply:
Removed unused code